processor/schemaprocessor/go.mod

```diff
@@ -5,7 +5,6 @@ go 1.17
 require (
 	github.com/stretchr/testify v1.7.1
 	go.opentelemetry.io/collector v0.50.1-0.20220429151328-041f39835df7
-	go.opentelemetry.io/collector/model v0.50.0
```

Comment:
I think this and a couple of other changes in unrelated components are unintentional? I think it is better to remove them to not pollute the PR

Reply:
that's a good point, this change just came from running `make gotidy` from the root after I rebased to resolve conflicts in `exporter/googlecloudexporter/go.mod` and `processor/schemaprocessor/go.sum`. Not sure where this came from

Reply:
I get the same diff running gotidy from `main` as well. Is this not supposed to be run with each change?

Reply:
Updated to remove the unrelated changes from this PR